Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM node:6
+FROM node:boron
 
 RUN apt-get update \
     && apt-get install -y \
         git \
+        # Next 2 needed for yarn
+        apt-transport-https \
```

Comment:
I can't comment on these Dockerfile settings. Does this file need additional separate review from devops (MattC or Michael)? (ok if they already signed off, otherwise a blocker for merge)